### PR TITLE
Add BrightID Verified User Registry contract

### PR DESCRIPTION
**Pull request by siftal**
This is a contract to register verified users context ids by BrightID node's verification data, and be able to query a user verification

### CODE REVIEW

File: contracts/contracts/verifiedUserRegistry/VerifiedUserRegistry.sol [hunk 1]
```diff
@@ -0,0 +1,91 @@
+pragma solidity ^0.6.0;
```
Comment (xuhcc) [inline contracts/contracts/verifiedUserRegistry/VerifiedUserRegistry.sol:1]: Currently we use solidity 0.5.17. Is it possible to compile this contract with this version?

File: contracts/contracts/verifiedUserRegistry/VerifiedUserRegistry.sol [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        verifications[_addrs[0]].time = _timestamp;
+        verifications[_addrs[0]].isVerified = true;
+        for(uint i = 1; i < _addrs.length; i++) {
+            // update time of all previous context ids to be sure no one can use old verifications again
+            verifications[_addrs[i]].time = _timestamp;
+            // set old verifications unverified
+            verifications[_addrs[i]].isVerified = false;
+        }
+    }
+}
```
Comment (xuhcc) [inline contracts/contracts/verifiedUserRegistry/VerifiedUserRegistry.sol:91]: Please fix linter errors: `yarn lint:contracts`

**Review by xuhcc — CHANGES_REQUESTED** [review]

**Comment by adamstallard** [comment]
@siftal Could you please add this event
```
event Sponsor(address);
```
and this function
```
 // sponsor any address provided as a parameter.
 function sponsor(address add) public {
   emit Sponsor(add);
 }
```
? That way we can also close this issue at the same time https://github.com/clrfund/monorepo/issues/136

File: contracts/contracts/verifiedUserRegistry/VerifiedUserRegistry.sol [hunk 3]
```diff
@@ -0,0 +1,91 @@
+pragma solidity ^0.6.0;
+
+import "./IVerifiedUserRegistry.sol";
+import "https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/access/Ownable.sol";
```
[the remark this answers is not in the document]
Comment (siftal) [reply contracts/contracts/verifiedUserRegistry/VerifiedUserRegistry.sol]: It's done.
https://github.com/siftal/monorepo/commit/faab2c83b3435b627f01809f1d65dcfda35b9c51

File: contracts/contracts/verifiedUserRegistry/VerifiedUserRegistry.sol (hunk 1 above)
Comment (siftal) [reply contracts/contracts/verifiedUserRegistry/VerifiedUserRegistry.sol]: Yes. I updated the compiler version.
https://github.com/siftal/monorepo/commit/7493a5cb6b1c4677abfff620c14a7ec8a82ae38f

File: contracts/contracts/verifiedUserRegistry/VerifiedUserRegistry.sol (hunk 2 above)
Comment (siftal) [reply contracts/contracts/verifiedUserRegistry/VerifiedUserRegistry.sol:100]: I used these libraries to check my contracts, but they didn't find any error
https://github.com/protofire/solhint
https://github.com/duaraghav8/Ethlint
What linter do you use?

Comment (xuhcc) [reply contracts/contracts/verifiedUserRegistry/VerifiedUserRegistry.sol:100]: `yarn lint:contracts` calls solhint with this config: https://github.com/clrfund/monorepo/blob/develop/contracts/.solhint.json

We use single quotes in our codebase: `error  Use single quotes for string literals  quotes`

**Review by xuhcc — APPROVED** [review]
Thanks @siftal !

**Pull request merged**